test/UnbrandedProjects/Platform-OpenAI-TypeSpec/src/Generated/Models/CreateCompletionResponseChoice.cs

```diff
@@ -37,7 +37,7 @@ internal CreateCompletionResponseChoice(long index, string text, CreateLogprobs
         /// <summary> Gets the text. </summary>
         public string Text { get; }
         /// <summary> Gets the logprobs. </summary>
-        public CreateLogprobs Logprobs { get; }
+        public CreateCompletionResponseChoiceLogprobs Logprobs { get; }
```

Comment:
This is union under anonymous model above: https://github.com/Azure/autorest.csharp/blob/37812dab10bf294ef00614f615e6a8df03f46100/test/UnbrandedProjects/Platform-OpenAI-TypeSpec/completions/models.tsp#L402